Content.Shared/Clothing/ClothingSpeedModifierSystem.cs

```diff
@@ -46,9 +48,12 @@ private void OnHandleState(EntityUid uid, ClothingSpeedModifierComponent compone
         component.WalkModifier = state.WalkModifier;
         component.SprintModifier = state.SprintModifier;
 
+        if (!_xformQuery.TryGetComponent(uid, out var xform))
```

Comment:
you can just use Transform(uid) to get the xform and you can assume it always has it
Transform does the query itself so it just saves on code duplication

Reply:
Already fixed on master